### DISCUSSION TITLE
Cancellazione voti

### DISCUSSION THREAD

**Discussion by sempliciniluigi89** [iisgiua/giuaschool#160] [Richieste di aiuto] [answered]
Buongiorno, sarebbe possibile implementare un tasto che permetta di cancellare le valutazioni di  tutti gli alunni senza dover fare alunno per alunno?

**Comment by trinko** [comment, marked as answer]
Nell'uso quotidiano del registro a scuola, non mi pare sia di grande utilità. 
Potrebbe essere utile per fare delle prove, ma se si tratta di prove, si fa prima a cambiare alunno o classe.

Vorrei essere chiaro: personalmente aggiungerò/modificherò solo le funzionalità che possono essere utili nel lavoro quotidiano della mia scuola. Per il resto, spero che qualcun altro prenda in carico queste segnalazioni e si occupi dello sviluppo del codice.
